WordPress/Classes/Services/ReaderCardService.swift

```diff
@@ -38,9 +38,14 @@ class ReaderCardService {
                                     page: self.pageHandle(isFirstPage: isFirstPage),
                                     refreshCount: refreshCount,
                                     success: { [weak self] cards, pageHandle in
-                guard let self = self else {
+                guard let self else {
                     return
                 }
+                var updatedCards = cards
+                if isFirstPage && updatedCards.first?.type == .interests && updatedCards.count > 2 {
+                    // Move the first tags recommendation card to a lower position
+                    updatedCards.move(fromOffsets: IndexSet(integer: 0), toOffset: 3)
```

Comment:
I almost thought that assigning `3` is a bug 😅 , if not for this note from the method documentation:

> If source represents a single element, calling this method with its own offset, or the offset of the following element, as destination has no effect.
> [Source](https://developer.apple.com/documentation/swift/mutablecollection/move(fromoffsets:tooffset:)#discussion)